.travis.yml

```diff
@@ -14,7 +14,7 @@ script:
   - pipenv install
   - pipenv install --dev --skip-lock
   - pipenv run nose2 -vvv tests.base_app_tests
-  - pipenv run nose2 -vvv --with-coverage
+  - pipenv run nose2 -vvv --with-coverage --cover-package=search
```

Comment:
We're using a ``.coveragerc`` file in the root of the repo to control coverage calculations. I tweaked this in 547e837 to focus on the code that we want to test. If it was including something else spurious we should update ``.coveragerc`` rather than using this flag. See https://coverage.readthedocs.io/en/v4.5.x/config.html for background.